### ISSUE TITLE
Breaking animation textures should be transparent

### ISSUE THREAD

**Issue by Moondarker** [PrismarineJS/prismarine-web-client#149]
Note that in game breaking process blends nicely with block texture

Current state:
![image](https://user-images.githubusercontent.com/4098364/112343442-2fedc880-8cd4-11eb-8de2-41a1fcd07d01.png)

Expected:
![image](https://user-images.githubusercontent.com/4098364/112343581-5449a500-8cd4-11eb-8239-9425259d56cc.png)

**Comment by Karang** [comment]
By the look of it, it seems to be a different blending mode and not simply a different oppacity. I've changed the blending mode to multiply which i think is close enough (but a bit too dark). There isn't really a way to control how much light/dark the cracks should be without editing the texture. So let me know if you consider it fixed.

**Comment by Moondarker** [comment]
Yep, it's better, but I'd like to keep the issue for now in case someone wants to further improve it :)
Might even try it myself later, gotta get familiar with three.js
